drivers/completer/completer.go

```diff
@@ -435,6 +435,10 @@ func (c completer) complete(previousWords []string, text []rune) [][]rune {
 	if TailMatches(IGNORE_CASE, previousWords, "FROM|JOIN") {
 		return c.completeWithSelectables(text)
 	}
+	/* TABLE <table> */
+	if TailMatches(IGNORE_CASE, previousWords, "TABLE") {
```

Comment:
```suggestion
	/* TABLE, but not TABLE embedded in other commands */
	if matches(IGNORE_CASE, previousWords, "TABLE") {
```
See how it's done in `psql`: https://github.com/postgres/postgres/blob/master/src/bin/psql/tab-complete.c#L4384

Reply:
Fixed.